include/EVT/dev/platform/f3xx/f302x8/RTC302x8.hpp

```diff
@@ -0,0 +1,55 @@
+#ifndef _EVT_RTCf302x8_
+#define _EVT_RTCf302x8_
+
+#include <HALf3/stm32f3xx.h>
+
+#include <EVT/dev/RTC.hpp>
+
+namespace EVT::core::DEV {
+
+
+/**
+ * RTC implementation that makes use of the on-board real time clock
+ * on the STM32f302x8.
+ *
+ * The onboard RTC can be kept up to date with a coin cell battery.
+ */
+class RTCf302x8: public RTC {
+public:
+
+    /**
+     * Empty constructor. Will startup the RTC.
+     */
+    RTCf302x8();
+
+    /**
+     * Get the current time as determined by the real time clock.
+     *
+     * @param[out] time The time struct to populate
```

Comment:
I haven't seen this `[out]` or `[in]` syntax before.  Is this used by sphinx?

Reply:
Its actually a C/C++ doc convention. In means the parameter is used as output. Out means the parameter is being used as an output. So in this example the time struct is actually storing the output of this function.

Reply:
> Its actually a C/C++ doc convention.

Will we be adopting this convention to the rest of the codebase?

Reply:
Ideally yes, just to clear up if an argument is used as an input, or if the caller should expect the parameter is store the output of the call.